src/main/java/com/bootcamp/demo/security/SecurityConfig.java

```diff
@@ -24,6 +24,8 @@ public void configure(final HttpSecurity http) throws Exception {
                 .and()
                 .logout()
                 .permitAll();
+        // temporal solution
+        http.csrf().disable();
```

Comment:
As discussed previously, here are the steps that you can follow in order to see how csrf could be enabled
- Google someting like `spring boot swagger post 403`
- The first page for results was this one: https://stackoverflow.com/questions/64837274/swagger-post-return-403-forbidden-spring-boot-spring-security
- Afterwards, you can search how to enable csrf (since above we saw that it is the cause for the issue), by googling something like: `spring boot swagger csrf enable`
- This led to having this `https://stackoverflow.com/questions/61956227/csrf-support-in-springdoc-openapi-swagger-ui` as first result
- Check what the option does & enable it if suitable & test if it works.

Reply:
I managed to make it work with csrf enabled in the browser with swagger. Only, I can't make the call from postman.

Reply:
Can you make the call from postman with the current config/setup?